get-started/setup-an-ide.md

```diff
@@ -1,7 +1,89 @@
 # Setup an IDE
 
-* vs code
-* goland
-* vim \(optional\)
-* emacs \(optional\)
+## vs code
 
+## GoLand
+
+You can use [GoLand](https://www.jetbrains.com/go/) to easily run or debug TiDB in many situations.
+
+> Goland uses [delve](https://github.com/go-delve/delve) to debug go programs, usually GoLand has an usable delve and needn't any additional configuration. But in some cases, you may need to download a delve by yourself.
+
+1. Make sure your `go` environment works well, see previous sections for more information.
+2. Open the TiDB project in GoLand.
+3. Add these config files to `<tidb-dir>/.idea/runConfigurations/` directory:
```

Comment:
seems like you should remove the heading spaces to render the content correctly

Reply:
do you mean the space between `.` and `M` in `1. M`?

I try to remove them but get worse format.